.buildkite/deploy/kubernetes/deploy-k8s-pipeline.yml

```diff
@@ -4,13 +4,17 @@ env:
   IMAGE_UBUNTU_X86_64: "family/platform-ingest-beats-ubuntu-2204"
   GCP_HI_PERF_MACHINE_TYPE: "c2d-highcpu-16"
 
+notify:
+  - github_commit_status:
+      context: "Deploy/K8S"
+
 steps:
   - label: "Checks"
     command: ".buildkite/deploy/kubernetes/scripts/make.sh"
-    if: build.env("BUILDKITE_PULL_REQUEST") != "false" && build.env("GITHUB_PR_LABELS") == "kubernetes"
+    if: build.env("BUILDKITE_PULL_REQUEST") != "false" && build.env("GITHUB_PR_LABELS") =~ /.*kubernetes.*/
```

Comment:
Can we add all the steps that executed on `kubernetes` label to one group with a single `if` condition? Just to avoid duplication